codecov.yml

```diff
@@ -31,6 +31,7 @@ codecov:
 ignore:
   - "**/*_generated.go"
   - "**/*_mock.go"
```

Comment:
```
❯ find . -type file -name '*_mock.go'

```
huh.  yea, prefix seems fine - helps correlate them all.

Reply:
mockery does not allow the setting of a file name and adds mock as a prefix when it is inpackage (for some unknown reason)

Reply:
`--filename` sets the filename.

it is kinda odd that it isn't used in the `mocks/` folder though.

Reply:
I missed that. I'll switch to _mock suffix for a unified approach.